frappe/www/printview.py

```diff
@@ -90,6 +90,8 @@ def get_rendered_template(
 	settings=None,
 ):
 
+	assert not print_format or not print_format.print_format_builder_beta
```

Comment:
We don't use asserts in production code. I am not much aware of what's going on here but this should ideally be replaced with a `frappe.throw` and an appropriate error message.

Reply:
I can understand, basically in the current form this function should never be called with the new style Print Format. With the fixes i made, it won't be called anymore. I can change it too frappe.throw, shall I do this?